ansible/plugins/vars/bw_plugin.py

```diff
@@ -0,0 +1,46 @@
+from ansible.plugins.vars import BaseVarsPlugin  # type:ignore[import]
```

Comment:
can you run black on this file (also in CI by adding it to `.github/workflows/lint.yml`)?